### DISCUSSION TITLE
코드 컴파일 결과 확인 도구

### DISCUSSION THREAD

**Discussion by bky373** [DaleStudy/leetcode-study#523] [자료공유]
저도 그렇고, 간혹 컴파일 결과를 궁금해하는 분들이 계셔서 공유드립니다.
- [Godbolt.org](https://godbolt.org/)

코드를 컴파일하고 어셈블리 코드를 생성해주는 웹사이트입니다. C, C++, Rust, Java 등 웬만한 프로그래밍 언어를 모두 사용할 수 있고, 코드를 입력하면 해당 코드가 어떻게 기계어로 번역되는지 확인할 수 있습니다. 코드 최적화나 다양한 컴파일러의 동작을 이해하는 데 도움이 될 것 같습니다.

예시)
<img width="1467" alt="image" src="https://github.com/user-attachments/assets/1d715e00-c766-4320-9b3e-68f782f45f52">

**Comment by obzva** [comment]
좋네요 ㅎㅎ 여러 옵션이 주어졌을 때, 각각 컴파일해보고 최적의 안을 선택하는데 유용할 것 같아요
